### PR TITLE
Selling the sizzle not the sausage

### PR DESCRIPTION
**Pull request by cwmyers**

### CODE REVIEW

File: lib/src/main/kotlin/app/cash/quiver/continuations/OutcomeEagerEffectScope.kt [hunk 1]
```diff
@@ -39,9 +39,6 @@ public value class OutcomeEffectScope<E>(private val cont: EffectScope<Either<Fa
       is Present -> value
     }
 
-//  public suspend fun ensure(value: Boolean): Unit =
-//    ensure(value) { None }
```
Comment (cwmyers) [inline lib/src/main/kotlin/app/cash/quiver/continuations/OutcomeEagerEffectScope.kt:43]: Left over when I was implementing `bind` on Outcome.

File: README.md [hunk 2]
```diff
@@ -5,6 +5,11 @@
 Quiver is a library that builds upon [Arrow](https://arrow-kt.io/) to make functional programming in Kotlin even 
 more accessible & delightful.
 
+Main features include:
```
Comment (cwmyers) [inline README.md:8]: I felt we needed some concrete benefits in the top level of the README

File: lib/src/main/kotlin/app/cash/quiver/continuations/OutcomeEagerEffectScope.kt [hunk 3]
```diff
@@ -29,7 +29,7 @@ value class OutcomeEagerEffectScope<E>(private val cont: EagerEffectScope<Either
 }
 
 @JvmInline
-public value class OutcomeEffectScope<E>(private val cont: EffectScope<Either<Failure<E>, Absent>>) :
```
Comment (cwmyers) [inline lib/src/main/kotlin/app/cash/quiver/continuations/OutcomeEagerEffectScope.kt:32]: Redundant modifier.  I don't think this is needed

**Review by hugomd — APPROVED** [review]
Great !

**Pull request merged**